Modules/mmapmodule.c

```diff
@@ -968,6 +976,7 @@ mmap_subscript(mmap_object *self, PyObject *item)
 
             if (result_buf == NULL)
                 return PyErr_NoMemory();
+
```

Comment:
Could the buffer have been invalidated here as a side effect of the PyMem_Malloc call above triggering GC? I seem to recall that's possible, but not 100% sure.

Reply:
I cannot find any GC-related code in `Object/obmalloc.c`